### PR TITLE
test: Add await

### PR DESCRIPTION
**Pull request by danieljbruce**
This should fix a flakey test issue where we wait for an instance to be created.

### CODE REVIEW

**Review by kolea2 — DISMISSED** [review]
please prefix this with "test" rather than "fix" so it does not show up in the release notes as a bugfix

**Comment by danieljbruce** [comment]
> please prefix this with "test" rather than "fix" so it does not show up in the release notes as a bugfix

Done!

File: samples/test/filters.js [hunk 1]
```diff
@@ -19,12 +19,14 @@ const snapshot = require('snap-shot-it');
...
 const {describe, it, before, after} = require('mocha');
 const cp = require('child_process');
-const {obtainTestInstance} = require('./util');
+const {obtainTestInstance, createTestInstance} = require('./util');
 
 const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
 const runId = uuid.v4().split('-')[0];
 const TABLE_ID = `mobile-time-series-${runId}`;
 
+createTestInstance();
```
Comment (bcoe) [inline samples/test/filters.js:28]: This will create an `unhandledRejection`, so I think we shouldn't do this in the top level.

**Comment by danieljbruce** [comment]
Closed in favour of moving `obtainTestInstance` instead.

**Pull request closed without merging**